docs/ERdiag.md

```diff
@@ -0,0 +1,31 @@
+```mermaid
+
+erDiagram
+    USER }|--o{ GROUP : composes
+    GROUP ||--o{ EXPENSE : contains
+    USER ||--o{ EXPENSE : makes
+    USER }|--|{ EXPENSE : "is concerned by"
+
+USER {
+    String username
+    String name 
+    String surname
+    String mail 
+    int phoneNumber 
```

Comment:
```suggestion
    String phoneNumber 
```

It depends on several things, and `int` can be a good type for `phoneNumber`, but it can create limitations in many cases. Maybe starting with `String` type can avoid you problems in the future ?

https://dba.stackexchange.com/questions/124546/datatype-for-phone-number-varchar-int-or-bigint